packages/react-vapor/src/components/drop/tests/Drop.spec.tsx

```diff
@@ -55,8 +58,8 @@ describe('Drop', () => {
             });
 
             const mountDropWithStore = (
-                props: Partial<IDropProps>,
-                store: mockStore<IReactVaporState>,
+                props?: Partial<IDropProps>,
+                store?: MockStoreEnhanced<IReactVaporState, IDispatch<IReactVaporState>>,

```

Comment:
What about exposing a `ReactVaporMockStore` type as well as `getStoreMock` ? Those will most likely be used together anyway :)

Reply:
I like the idea, I'll make the change